server/views/partials/navbar.jade

```diff
@@ -3,18 +3,16 @@ nav.navbar.navbar-default.navbar-fixed-top.nav-height
         button.hamburger.navbar-toggle(type='button', data-toggle='collapse', data-target='.navbar-collapse')
             .col-xs-12
                 span.hamburger-text Menu
-        a.navbar-brand(href='/challenges/next-challenge')
-            img.img-responsive.nav-logo(src='https://s3.amazonaws.com/freecodecamp/freecodecamp_logo.svg', alt='learn to code javascript at Free Code Camp logo')
+        a.navbar-brand(href='/challenges/next-challenge' rel='nofollow')
```

Comment:
Good question. I looked into this further and it seems this won't help according to a Google search engineer.